chore: update IPFS Desktop testing steps (#8393)
ipfs/go-ipfs: docs/RELEASE_ISSUE_TEMPLATE.md

    - [ ] Deploy new version to a subset of Preload nodes
    - [ ] Collect metrics every day. Work with the Infrastructure team to learn of any hiccup
  - [ ] IPFS Application Testing -  Run the tests of the following applications:
    - [ ] [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop)

>       - [ ] Ensure the RC is published to [the NPM package](https://www.npmjs.com/package/go-ipfs?activeTab=versions)

Reviewer: ```suggestion
      - [ ] Ensure the RC is published to [the NPM package](https://www.npmjs.com/package/go-ipfs?activeTab=versions) ([happens automatically, just wait for CI](https://github.com/ipfs/npm-go-ipfs/actions))
```

  - [ ] Merge the `release` branch back into `master`, ignoring the changes to `version.go` (keep the `-dev` version from master).
  - [ ] Create an issue using this release issue template for the _next_ release.
  - [ ] Make sure any last-minute changelog updates from the blog post make it back into the CHANGELOG.

>   - [ ] Merge the PR created for IPFS Desktop into main

Reviewer: I think merge is out of scope – marking PR as "ready" makes a better checklist item. 
```suggestion
  - [ ] Mark PR draft created for IPFS Desktop as ready for review.
```
